### PR TITLE
test: add tests for SSR

### PR DESCRIPTION
**Pull request by Brailor**
Add simple test cases for testing whether our components can be rendered on the server side with node.js (thus the `testEnvironment='node'`).

### CODE REVIEW

File: packages/ui/__tests__/ui.spec.js [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+  }
+}
+
+describe('Testing every imported component for SSR', () => {
+  beforeAll(() => {
+    // since we don't provide every needed prop propTypes will complain when
+    // we render the components, so in order to not to spam the console
+    // we supress these warnings/errors
+    jest.spyOn(console, 'warn').mockImplementation(() => {})
+    jest.spyOn(console, 'error').mockImplementation(() => {})
```
Comment (matyasf) [inline packages/ui/__tests__/ui.spec.js:253]: I think you need to call `mockRestore` after the test has run so it does not influence other tests https://jestjs.io/docs/mock-function-api#mockfnmockrestore

**Review by matyasf — CHANGES_REQUESTED** [review]
I think mocks need to be restored

File: packages/ui/__tests__/ui.spec.js (hunk 1 above)
Comment (Brailor) [reply packages/ui/__tests__/ui.spec.js:253]: Thanks, I changed the `restoreMocks` config to be true.

**Review by matyasf — APPROVED** [review]
looks good :)

**Review by matyasf — APPROVED** [review]

**Pull request merged**